index.html

```diff
@@ -1438,8 +1437,8 @@ <h2>Verification Methods</h2>
 Verification methods might take many parameters. An example of this is a set
 of five cryptographic keys from which any three are required to contribute to
 a threshold signature. Methods need not be cryptographic. An example of this
-might be the contact information for a biometric service provider that compares
-a purported <a>DID controller</a> against a candidate biometric vector.
+might be the contact information for a verifiable credential verifier that
```

Comment:
```suggestion
might be through the usage of a service endpoint that can be used to fetch a verifiable credential from the <a>DID Controller</a> so that the <a>DID Controller</a> can provide proof of control via a verifiable credential.
```

Reply:
@rhiaro this is an attempt to reword it. It still feels a bit messy to me, but thought it might help nudge you towards what you were describing.